_config.yml

```diff
@@ -6,7 +6,7 @@ highlighter:      rouge
 permalink:        pretty
 
 # Setup
-title:            'OSCAR Computer Algebra System'
+title:            'The Computer Algebra System OSCAR. Algorithms and Examples'
```

Comment:
I am wondering if this sounds too much like the OSCAR system website.

Reply:
This is the official title of the book, as far as I know.

Reply:
Right. But does the website have to have the same name? Maybe people get confused what website is for the system (e.g. release notes, installation instructions etc.) and which gives details about the corresponding spring book?